packages/transcript-view/src/transcript-view.ts

```diff
@@ -0,0 +1,383 @@
+        <duration-formatter .seconds=${this.currentEntryStartTime}> </duration-formatter>
+      </div>
+    `;
+  }
+
+  private transcriptEntryTemplate(entry: TranscriptEntryConfig): TemplateResult {
+    const currentEntryId = this.currentEntry ? this.currentEntry.id : -1;
+    const active = entry.id === currentEntryId;
+    const selected = entry.searchMatchIndex === this.selectedSearchResultIndex;
+    return html`
+      <transcript-entry
+        .entry=${entry}
+        ?isSelected=${selected}
+        ?isActive=${active}
+        data-search-result-index=${entry.searchMatchIndex}
+        @userSelected=${this.transcriptEntrySelected}
+      >
+      </transcript-entry>
+    `;
+  }
+
+  private get contextZoneDevTemplates(): TemplateResult {
+    return html`
+      ${this.topContextZoneDevTemplate} ${this.bottomContextZoneDevTemplate}
+    `;
+  }
+
+  private get topContextZoneDevTemplate(): TemplateResult {
+    return html`
+      <div class="top context-overlay" style="height: ${this.topContextHeight}px"></div>
+    `;
+  }
+
+  private get bottomContextZoneDevTemplate(): TemplateResult {
+    return html`
+      <div class="bottom context-overlay" style="height: ${this.bottomContextHeight}px"></div>
+    `;
+  }
+
+  get currentEntryStartTime(): number {
+    return this.currentEntry ? this.currentEntry.start : 0;
+  }
+
+  static get styles(): CSSResult {
+    const transcriptHeightCss = css`var(--transcriptHeight, 200px)`;
+
+    const timeColorCss = css`var(--timeColor, white)`;
+    const timeColumnWidthCss = css`var(--timeColumnWidth, 5rem)`;
+
+    const autoScrollButtonFontColorCss = css`var(--autoScrollButtonFontColor, black)`;
+    const autoScrollButtonBackgroundColorCss = css`var(--autoScrollButtonBackgroundColor, white)`;
+
+    return css`
+      .container {
+        position: relative;
+      }
+
+      .auto-scroll-button {
+        position: absolute;
+        left: 0;
+        right: 0;
+        margin: auto;
+        width: 8rem;
+        bottom: 1rem;
+        border-radius: 1rem;
+        border: 0;
+        color: ${autoScrollButtonFontColorCss};
+        background-color: ${autoScrollButtonBackgroundColorCss};
+      }
+
+      .context-overlay {
+        position: absolute;
+        left: 0;
+        width: 100%;
+        height: 0;
+        z-index: -1;
+        display: block;
+      }
+
+      .context-overlay.top {
+        top: 0;
+        border-bottom: 1px solid green;
+      }
+
+      .context-overlay.bottom {
+        bottom: 0;
+        border-top: 1px solid green;
+      }
+
+      .time {
+        flex: 0 0 ${timeColumnWidthCss};
+        color: ${timeColorCss};
+        position: relative;
+      }
+
+      .time-display {
+        position: absolute;
+        top: 0;
+      }
+
+      .scroll-container {
+        display: flex;
+        overflow-y: auto;
+        font-size: 16px;
+        line-height: 24px;
```

Comment:
we have been using `rem` in PB. perhaps we can revisit this as a collective

Reply:
That may be difficult since the idea is to get the same default display from this component no matter where you render it. If we're working with a base value of `20px`, for example, setting this to `font-size: 1.6rem;` would give us a font-size of 32 rather than 16.

Reply:
Oh yeah actually let me make these CSS variables that can be configured from the consumer. I think I set these when Jim and I were trying to calculate the line height, but we don't need to any more.